src/modules/views/Folder/ExtraDrive.tsx

```diff
@@ -52,12 +62,15 @@ export const ExtraDrive = (): JSX.Element | null => {
       >
         {files.map(file => (
           <NavItem
-            key={file._id}
+            key={file.id}
```

Comment:
Shouldn't we always use _id ? 🤔

Reply:
well, I had an item without `_id`, not sure how that happened though? But looking in the project, `file.id` is used 40 times, and `file._id` 11 times. Is this something to investigate?

Reply:
I don't know how you can have a file without an `_id` since this is what we store to the DB. In fact cozy-client is there to let you use `_id` or `id` so you should be able to use both.